x-pack/plugins/apm/server/routes/agent_explorer/fetch_agents_latest_version.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { Logger } from '@kbn/core/server';
+import { isEmpty } from 'lodash';
+import { AgentName } from '../../../typings/es_schemas/ui/fields/agent';
+import { fetchWithTimeout } from '../../lib/helpers/fetch_with_timeout';
+
+export interface ElasticAgentLatestVersion {
```

Comment:
To avoid confusion with Elastic Agent:

```suggestion
export interface ElasticApmAgentLatestVersion {
```

Reply:
Addressed in [b677f3e](https://github.com/elastic/kibana/pull/153643/commits/b677f3efad3b6fb0f42f02ea2bea4e6787419145).